proxy/src/main/resources/default-velocity.toml

```diff
@@ -15,8 +15,18 @@ show-max-players = 500
 # Should we authenticate players with Mojang? By default, this is on.
 online-mode = true
 
-# Should the proxy enforce the new public key security standard? By default, this is on.
-force-key-authentication = true
+# Should the proxy support secure profiles (and signed chat) for 1.19.1+ clients and servers?
+# This option only affects 1.19.1+ clients and servers.
+#
+# Available options:
+# - "disabled":    Secure profiles are not checked, all chat messages will appear unsigned to backend
+#                  servers. This option is not recommended as it may break compatibility with 1.19.1
+#                  servers, and it will cause newer clients to show a warning upon login.
```

Comment:
Should this option allow cancelling signed chat messages on 1.19.1+? I tested this branch on a local server, but it still kicks me when sending any message on 1.19.1. This means that I can't convert everything to a system message (through a global chat plugin for example)

Reply:
What error are you getting? It should work if you set it to `disabled`.

Reply:
Same error as before where it says canceling chat messages isn't supported on 1.19.1+, and it kicks the player. Maybe [this if statement](https://github.com/PaperMC/Velocity/blob/cae41c33a1a758f5ecd0a4361c04e8652dd67072/proxy/src/main/java/com/velocitypowered/proxy/connection/client/ClientPlaySessionHandler.java#L213) needs to be modified

Reply:
When key auth is disabled, `player.getIdentifiedKey()` is null. I can cancel messages just fine, maybe you need to use `DISABLED`? (in caps)

Reply:
Works completely fine on my end. When disabled, messages can be blocked. Otherwise, the killswitch gets triggered as the Velocity devs intended.